### ISSUE TITLE
Download documentation needs updating, --repo_id required

### ISSUE THREAD

**Issue by Sciumo** [Lightning-AI/lit-parrot#44]
Document says:
```bash
python scripts/download.py stabilityai/stablelm-base-alpha-3b
```
Actually required
```bash
python scripts/download.py --repo_id stabilityai/stablelm-base-alpha-3b
```
I could just make edits as I go along and send PR if you wish.

**Comment by carmocca** [comment]
Please do! You can also set `as_positional=False` in https://github.com/Lightning-AI/lit-parrot/blob/main/scripts/download.py#L35 and then it won't require `--repo_id`

**Comment by Sciumo** [comment]
Okay.  I'll include some sanity and a fix as well.
Not sure how convert_hf_checkout.py ever worked since the  check_valid_checkpoint_dir looks for lit_model.pth and lit_config.json doesn't even exist yet.

**Comment by carmocca** [comment]
This was fixed in #48. Thank you for reporting

**Comment by marimeireles** [comment]
Hey @carmocca not sure if this was fixed, I'm seeing the same issue by following lightning's [tutorial](https://lightning.ai/pages/blog/how-to-finetune-gpt-like-large-language-models-on-a-custom-dataset/). Checkpoints doesn't exist at the point is being used.
